app/scripts/lib/mailcheck.js

```diff
@@ -33,21 +30,14 @@ function (Tooltip, Url) {
    * @param {Object} target DOM input node to target with mailcheck
    * @param {Object} metrics Metrics logger
    * @param {Object} translator Translator object passed by the view
-   * @param {String} queryParams Window query params
    */
-  return function checkMailInput(target, metrics, translator, queryParams) {
+  return function checkMailInput(target, metrics, translator) {
     var element = $(target);
-    if (!element.length) {
+    if (! element.length) {
       return;
     }
-    var enabled = false;
-
-    // Url param flag 'mailcheck' to disable the feature unless the flag is set
-    if (Url.searchParam('mailcheck', queryParams) === '1') {
-      enabled = true;
-    }
     // check if the text value was changed before showing the tooltip
-    if (element[0].previousValue !==  element.val() && element.val().length > MIN_CHARS && enabled) {
+    if (element[0].previousValue !== element.val() && element.val().length > MIN_CHARS) {
```

Comment:
Oh, I didn't notice this before, `previousValue` is an expando property on the element. data attributes, or [$(&lt;element&gt;).data](http://api.jquery.com/data/) can be used for that purpose too w/o adding properties to the element itself.